### ISSUE TITLE
feat(patch): Fix Descriptions

### ISSUE THREAD

**Issue by egscodes** [revanced/revanced-patches#2037]
### Application

Youtube

### Issue

Some of the patches add a toggle but in the description they don't mention it, they just say it removes stuff

### Patch

Description should say "Adds a toggle that ____" (or any variation depending on the context)

This is a list of some of the patches that i use that have a toggle but don't mention them, there's probably more but i don't want to repatch my current apk just to test

- Client spoof (i think)
- Disable auto caption
- Disable shorts on startup
- Disable zoom haptics
- Downloads
- Hdr auto brightness
- Hide album cards
- Hide Artist card
- Hide autoplay button
- Hide breaking news shelf
- Hide create button
- Hide crowdfunding box
- Hide short button
- Hide watch in vr
- hide watermark
- minimized playback
- old quality layout
- open links externally
- seekbar tapping
- spoof app version

### Motivation

It will make it easy to know what adds a toggle and what simply just works.

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by oSumAtrIX** [comment]
The descriptions should be short, can you please suggest short descriptions for every of the toggles you mentioned?

**Comment by egscodes** [comment]
Looking at the descriptions one could either say "Toggle that ----" or it could be written like the Hide player buttons "**Adds the option to** hide video player previous and next buttons."

Example with **disable-auto-captions** "Disable forced captions from being automatically enabled."
1. Adds the option to Disable forced captions from being automatically enabled.
2. Adds Toggle that Disables forced captions from being automatically enabled. (one can even remove the word Adds)

**Comment by oSumAtrIX** [comment]
@LisoUseInAIKyrios I believe the issue is in the descriptions of the toggle preferences. They just say "enabled" or "disabled" expressing the state of the toggle, whereas the description is meant to describe the toggle, not just express its state.

**Comment by oSumAtrIX** [comment]
> Of course, let the user manually include/exclude patches if they wish, but make that an 'expert' mode or something to that effect.

Yes, this is in ongoing discussion in ReVanced Manager repository. The idea is to promote the default selection of patches but allow the user to alter it if they really want to.

**Comment by oSumAtrIX** [comment]
The problem is something like "Hide product banner" does not explain what the product banner is. People don't know what they are hiding.

**Comment by SodaWithoutSparkles** [comment]
Maybe do a screenshot example on what that actually hides

**Comment by SodaWithoutSparkles** [comment]
> There is no room for screenshots in the components and design language that YouTube uses.

I mean in some documentation repo

**Comment by oSumAtrIX** [comment]
All documentation is agnostic to any patch. At max patches can document itself
